thorlcr/thorutil/thmem.cpp

```diff
@@ -451,7 +428,7 @@ const void *CThorExpandingRowArray::allocateRowTable(rowidx_t num, unsigned maxS
 
 rowidx_t CThorExpandingRowArray::getNewSize(rowidx_t requiredRows)
 {
-    rowidx_t newSize = maxRows;
```

Comment:
Well.. it was, but wrong.
It was so it had any effect when reducing in size (ignoring existing maxSize). 
However, I shouldn't have used this function which is for expanding from maxRows size only.

Will fix.